views/layout.pug

```diff
@@ -44,4 +44,8 @@ html
 				a.c-footer-icon-link(href="https://github.com/sahil505/StyleGuideDesigner", target="_blank")
 					svg.o-icon.c-footer-icon
 						use(xlink:href="#github")
-				p.c-footer-text Developed for <a href="https://www.phpbb.com/" target="_blank">phpBB</a>
\ No newline at end of file
+				p.c-footer-text Developed for <a href="https://www.phpbb.com/" target="_blank">phpBB</a>
+				script(src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js")
+				script(src="https://unpkg.com/vue@2.0.3/dist/vue.js")
```

Comment:
Still not correct you have indented it into the footer! This is directly in the body no wrapping!